internal/engine/image_build_and_deployer.go

```diff
@@ -225,19 +225,7 @@ func (ibd *ImageBuildAndDeployer) deploy(ctx context.Context, st store.RStore, p
 // We can also skip the push of the image if it isn't used
 // in any k8s resources! (e.g., it's consumed by another image).
```

Comment:
I still don't think this comment applies to the whole function, and should therefore be either a. more complete or b. live INSIDE the func rather than at the top level

Reply:
updated!